edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/EquipmentTO.java

```diff
@@ -7,11 +7,13 @@
 
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
+import org.dcsa.skernel.infrastructure.validation.ISO6346EquipmentReference;
 
 @AllOrNone({"tareWeight", "weightUnit"})
 public record EquipmentTO(
   @NotNull(message = "EquipmentReference is required.")
   @Size(max = 15)
```

Comment:
You might as well change size to be `min=11, max=11` here as `@ISO6346EquipmentReference` has those size restrictions (or remove `@Size` and let `@ISO6346EquipmentReference` deal with it)`.

Reply:
Point taken `@Size` removed